src/api/app/helpers/webui/project_helper.rb

```diff
@@ -122,4 +122,11 @@ def escape_list(list)
         "']".html_safe
     end.join(',').html_safe
   end
+
+  def show_package_actions?
+    return false if @is_maintenance_project
+    return false if @project.defines_remote_instance?
+    return true unless @is_incident_project && @packages.present? &&
+                       @has_patchinfo && @open_release_requests.empty?
```

Comment:
[nitpick]

It's possible that `show_package_actions?` returns `nil`. It's not an issue as it is falsy when using it in a view. I think we should add a `false` after the last guard clause (so a fourth line in that method).

Reply:
I think you should just remove the `return true` and then it will already return true or false.

Reply:
@dmarcoux @Ana06 I personally would prefer to add ```return false```. Just for readability.